pom.xml

```
@@ -8,7 +8,7 @@
 		<version>2.2.0</version>
 	</parent>
 	<artifactId>data-model-lib</artifactId>
-	<version>1.9.0-SNAPSHOT</version>
+	<version>1.10.0-SNAPSHOT</version>
```

I think this is only a small bug fix and not a whole feature? So 
```suggestion
	<version>1.9.1-SNAPSHOT</version>
```

---

jeah, but it introduces an Exception, that might lead to unexpected behaviour in existing implementations. According to semantic versioning i was not 100% sure if a patch release is correct.

---

It should not. Existing implementations always have at least one item and are thus not affected.